cvrf2csaf/common/utils.py

```diff
@@ -43,7 +43,7 @@ def get_config_from_file() -> dict:
         with open(path_to_conf, 'r') as f:
             config = yaml.safe_load(f)
 
-        for key in ['force', 'force_update_revision_history']:
+        for key in ['force', 'force_insert_current_version_into_revision_history']:
```

Comment:
@sustefil This should be a `fix` as we are only doing this, if the current version is missing... If it would be named `force`, I would expect that it is done regardless of whether it is missing...

Reply:
Resolved, sorry I have probably misinterpreted this, thinking that --force prefix is ok.

"CSAF uses the terminology of quick-fixes for recommended actions if one of the tests from section 6 fails. That's why I came up with fix_current_version_not_in_revision_history. However, I'm also happy with a --force~ if it conveys that it adds only the current version if it is missing."